fdbserver/DataDistributionQueue.actor.cpp

```diff
@@ -1676,7 +1682,9 @@ ACTOR Future<Void> dataDistributionQueue(Database cx,
 					    .detail("PriorityTeam1Left", self.priority_relocations[SERVER_KNOBS->PRIORITY_TEAM_1_LEFT])
 					    .detail("PriorityTeam0Left", self.priority_relocations[SERVER_KNOBS->PRIORITY_TEAM_0_LEFT])
 					    .detail("PrioritySplitShard", self.priority_relocations[SERVER_KNOBS->PRIORITY_SPLIT_SHARD])
-					    .trackLatest("MovingData");
+					    .trackLatest("MovingData"); // this track event's lifestyle is contolled by
```

Comment:
A minor wording change suggestion:

> This trace event's trackLatest lifetime is controlled by DataDistributorData::movingDataEventHolder. The track latest key we use here must match the key used in the holder.

Reply:
A similar wording could be used in the other locations we have an analogous comment.